openassessment/templates/openassessmentblock/oa_rubric.html

```diff
@@ -55,15 +56,16 @@ <h4 class="question__title ui-toggle-visibility__control">
             </div>
         </li>
         {% endfor %}
-        <li class="wrapper--input field field--textarea assessment__rubric__question assessment__rubric__question--feedback" id="assessment__rubric__question--feedback">
-            <label class="question__title" for="assessment__rubric__question--feedback__value">
-                <span class="question__title__copy">{{ rubric_feedback_prompt }}</span>
+        <li class="wrapper--input field field--textarea assessment__rubric__question assessment__rubric__question--feedback">
+            <label class="question__title" for="{{ rubric_type }}__assessment__rubric__question--feedback__value">
+                <span class="question__title__copy">{% trans rubric_feedback_prompt %}</span>
```

Comment:
I'm not 100% sure about this, but this is the idiom used elsewhere in this file and others. The question to my mind is whether the string being passed as a parameter to this template will be processed by the string scraper. My intuition says it wouldn't be, but I don't know for certain yet. I think we probably need an i18n clean up story for ORA before we try to land the feature.